### PR TITLE
fix typo and add filesystem=xdg-run/keyring in manifest

### PR DESCRIPTION
**Pull request by ganthern**

### CODE REVIEW

File: manifest-template.js [hunk 1]
```diff
@@ -4,84 +4,84 @@ const fs = require('fs')
...
-    "branch": "stable",
-    "sdk": "org.freedesktop.Sdk",
-    "command": "tutanota-desktop",
-    "separate-locales": false,
-    "finish-args": [
-        "--share=ipc",
-        "--socket=x11",
-        "--socket=wayland",
-        "--socket=pulseaudio",
-        "--socket-system-bus",
```
Comment (charlag) [inline manifest-template.js:21]: aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa

how did I do this

File: manifest-template.js [hunk 2]
```diff
@@ -4,84 +4,84 @@ const fs = require('fs')
...
+	"finish-args": [
+		"--share=ipc",
+		"--socket=x11",
+		"--socket=wayland",
+		"--socket=pulseaudio",
+		"--socket=system-bus",
+		"--socket=session-bus",
+		"--share=network",
+		"--filesystem=host",
+		"--filesystem=xdg-run/keyring",
```
Comment (charlag) [inline manifest-template.js:24]: where is this documented?

File: manifest-template.js [hunk 3]
```diff
@@ -4,84 +4,84 @@ const fs = require('fs')
 const [hash, url] = process.argv.slice(2)
 
 const manifest = {
-    "app-id": "com.tutanota.Tutanota",
-    "runtime": "org.freedesktop.Platform",
-    "runtime-version": "21.08",
-    "base": "org.electronjs.Electron2.BaseApp",
-    "base-version": "21.08",
-    "branch": "stable",
```
[the remark this answers is not in the document]
Comment (ganthern) [reply manifest-template.js:12]: it's not documented and if it refers to the base app stable branch, that's 4 years old. 

I don't think it was used (the build logs seem to say ElectronBaseApp 21.08 was used), so I removed it to see if something breaks.

File: manifest-template.js (hunk 1 above)
Comment (ganthern) [reply manifest-template.js:21]: rather ask how that's not an error that breaks builds

File: manifest-template.js (hunk 2 above)
Comment (ganthern) [reply manifest-template.js:24]: couldn't find out what xdg-run does in general (maybe it's a special flatpak syntax?)
some examples of `--filesystem=xdg-run\*` are documented here, but not the keyring one: https://docs.flatpak.org/en/latest/sandbox-permissions.html

also see
https://github.com/flathub/com.github.vladimiry.ElectronMail/blob/master/com.github.vladimiry.ElectronMail.yaml

Comment (charlag) [reply manifest-template.js:24]: :+1:

**Pull request merged**